### PR TITLE
feat(watch): add `--clear-screen` flag

### PR DESCRIPTION
**Pull request by patheticGeek**
Related issue #25 

When running in watch mode, sometimes the user doesn't want the console to be cleared, so added this flag.
Plz accept.

### CODE REVIEW

**Comment by privatenumber** [comment]
Can you open an issue and explain your use-case?

**Comment by patheticGeek** [comment]
> Can you open an issue and explain your use-case?

Here, #25

File: src/watch.ts [hunk 1]
```diff
@@ -42,7 +47,11 @@ export const watchCommand = command({
 			runProcess.kill();
 		}
 
-		process.stdout.write(clearScreen);
+		if (!options.noClearScreen) {
+			process.stdout.write(clearScreen);
+		} else {
+			console.log('🔄 tsx: restarting');
```
Comment (knpwrs) [inline src/watch.ts:53]: It might be cool to make `noClearScreen` a boolean or string to provide a custom message here.

**Review by privatenumber — CHANGES_REQUESTED** [review]
Let's follow Vite's footsteps here: https://vitejs.dev/config/#clearscreen

Can you rename it to `clearScreen` and default it to `true`?

**Comment by patheticGeek** [comment]
> Let's follow Vite's footsteps here: https://vitejs.dev/config/#clearscreen
> 
> Can you rename it to `clearScreen` and default it to `true`?

Yep will do it!

**Comment by patheticGeek** [comment]
@privatenumber do you wanna change the message it logs?
Also gonna make it not log restarting on the first run cause thats start not a restart

**Comment by ccorcos** [comment]
This flag doesn't appear in the `--help` documentation btw.

**Pull request merged**